components/ws-manager/pkg/manager/status.go

```diff
@@ -595,7 +595,7 @@ func extractFailure(wso workspaceObjects) (string, *api.WorkspacePhase) {
 					// default way for headless workspaces to be done
 					return "", nil
 				}
-				return fmt.Sprintf("container %s completed; containers of a workspace pod are not supposed to do that", cs.Name), nil
+				return fmt.Sprintf("container %s completed; containers of a workspace pod are not supposed to do that. error msg: %s", cs.Name, terminationState.Message), nil
```

Comment:
Nit: considering that the error is user-facing, `error msg` doesn't sit quite right :)

```suggestion
				return fmt.Sprintf("container %s completed; containers of a workspace pod are not supposed to do that. Reason: %s", cs.Name, terminationState.Message), nil
```

Reply:
Good point! :heart: